tools/slo-generator/tests/unit/test_cli.py

```diff
@@ -0,0 +1,28 @@
+import os
+import unittest
+
+from slo_generator.cli import parse_args
+
+cwd = os.path.dirname(os.path.abspath(__file__))
+
+class TestCLI(unittest.TestCase):
+
+    def test_parse_args(self):
+        slo_config_path = f'{cwd}/fixtures/slo_config.json'
```

Comment:
standardize usage of f-strings throughout this asset (earlier you are using `"{}".format(val)`)

Reply:
done, thanks